yarn.lock

```diff
@@ -953,6 +953,136 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
+"@commitlint/cli@^8.3.5":
+  version "8.3.5"
+  resolved "https://registry.yarnpkg.com/@commitlint/cli/-/cli-8.3.5.tgz#6d93a3a8b2437fa978999d3f6a336bcc70be3fd3"
+  integrity sha512-6+L0vbw55UEdht71pgWOE55SRgb+8OHcEwGDB234VlIBFGK9P2QOBU7MHiYJ5cjdjCQ0rReNrGjOHmJ99jwf0w==
+  dependencies:
+    "@commitlint/format" "^8.3.4"
+    "@commitlint/lint" "^8.3.5"
+    "@commitlint/load" "^8.3.5"
+    "@commitlint/read" "^8.3.4"
+    babel-polyfill "6.26.0"
+    chalk "2.4.2"
+    get-stdin "7.0.0"
+    lodash "4.17.15"
+    meow "5.0.0"
```

Comment:
> This meow dependency is bringing in more of the [bad yargs-parser versions](https://www.npmjs.com/advisories/1500). I'm seeing at least one commitlint/cli PR attempting to address this, so it may get patched soon.
> 
> `standard-version` suffers the same issue, though it's more indirect (through `conventional-recommended-bump` and `mocha`). This one doesn't have an open issue or PR yet, so may take longer to get patched if we don't engage.

this was mentioned in our check-in today, but for posterity: we're going to add an ADR around our stance re: merging in dependencies that have open vulnerabilities, but tldr is that they should not block work, especially if low severity, and especially if in `devDependencies` (which won't end up in production code)